### PR TITLE
Cadence plugins now public

### PR DESCRIPTION
**Pull request by harrisonliew**

### CODE REVIEW

File: doc/CAD-Tools/Hammer-CAD-Tools.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Hammer CAD Tools
 ===============================
 
-Hammer currently has three repos for CAD tools from different vendors: ``hammer-cadence-plugins``, ``hammer-synopsys-plugins``, and ``hammer-mentor-plugins``. These repos are private since they contain commands specific to each tool, but access to them may be granted for Hammer users pending approval from their respective companies.  
+Hammer currently has three repos for CAD tools from different vendors: ``hammer-cadence-plugins``, ``hammer-synopsys-plugins``, and ``hammer-mentor-plugins``. ``hammer-cadence-plugins`` is a public repo but the others are private since they contain tool-specific commands not yet cleared for public release. Access to them may be granted for Hammer users who already have licenses for those tools. See the note about :ref:`plugins access <plugins-access>` for instructions for how to request access.
```
Comment (edwardcwang) [inline doc/CAD-Tools/Hammer-CAD-Tools.rst:6]: They still have to request right?

File: doc/Hammer-Basics/Hammer-Overview.rst [hunk 2]
```diff
@@ -47,6 +47,7 @@ The TCL commands input to the tool are created using technology and design setti
 
 There are already three Hammer tool plugin repos to which access may be granted to Hammer users. These repos are ``hammer-cadence-plugins``, ``hammer-synopsys-plugins``, and ``hammer-mentor-plugins``. In them are tool plugin implementations for actions including synthesis, place-and-route, DRC, LVS, and simulation.
```
Comment (dpgrubb13) [inline doc/Hammer-Basics/Hammer-Overview.rst:48]: Note here which are public and private.

Comment (harrisonliew) [reply doc/Hammer-Basics/Hammer-Overview.rst]: Done.

**Review by dpgrubb13 — APPROVED** [review]
LGTM

**Pull request merged**